app/javascript/dashboard/helper/snoozeHelpers.js

```diff
@@ -66,3 +66,23 @@ export const snoozedReopenTime = snoozedUntil => {
   }
   return snoozedUntil ? format(date, 'd MMM, h.mmaaa') : null;
 };
+export const snoozedReopenTimeToTimestamp = snoozedUntil => {
+  return snoozedUntil ? getUnixTime(new Date(snoozedUntil)) : null;
+};
+export const shortenSnoozeTime = snoozedUntil => {
+  if (!snoozedUntil) {
+    return null;
+  }
+  const shortenTime = snoozedUntil.replace(
+    /( minutes?| hours?| days?| months?| years?)/g,
+    (match, unitMatch) => {
+      if (unitMatch === ' minutes' || unitMatch === ' minute') return 'm';
+      if (unitMatch === ' hours' || unitMatch === ' hour') return 'h';
+      if (unitMatch === ' days' || unitMatch === ' day') return 'd';
+      if (unitMatch === ' months' || unitMatch === ' month') return 'mo';
+      if (unitMatch === ' years' || unitMatch === ' year') return 'y';
+      return match;
+    }
+  );
```

Comment:
```suggestion
export const shortenSnoozeTime = snoozedUntil => {
  if (!snoozedUntil) {
    return null;
  }

  const unitMap = {
    minutes: 'm',
    minute: 'm',
    hours: 'h',
    hour: 'h',
    days: 'd',
    day: 'd',
    months: 'mo',
    month: 'mo',
    years: 'y',
    year: 'y',
  };

  const shortenTime = snoozedUntil.replace(
    /\s(minute|hour|day|month|year)s?\b/gi,
    (match, unit) => unitMap[unit.toLowerCase()] || match
  );
};
```

This uses a map, instead of if-else cases to replace the unit, reducing the complexity, also, the extra blank space is handled in the regex pattern.